sdk-blueprints/python/post_process.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+
+
+def process_file(file_path):
+    file_name = os.path.basename(file_path)
```

Comment:
When the name of the function is really generic, please add a short function comment

Reply:
updated the name and added comment